Userland/Libraries/LibJS/Runtime/TypedArrayConstructor.cpp

```diff
@@ -121,13 +116,10 @@ JS_DEFINE_NATIVE_FUNCTION(TypedArrayConstructor::from)
     for (size_t k = 0; k < length; ++k) {
         auto k_value = array_like->get(k);
         Value mapped_value;
-        if (map_fn) {
-            mapped_value = vm.call(*map_fn, this_arg, k_value, Value(k));
-            if (vm.exception())
-                return {};
-        } else {
+        if (map_fn)
+            mapped_value = TRY_OR_DISCARD(vm.call(*map_fn, this_arg, k_value, Value(k)));
+        else
```

Comment:
I have no idea how but somehow this exposed that we are missing an exception check on the `array_like->get(k)` above.